samples/oauth_rails_vetverification/README.md

```diff
@@ -0,0 +1,24 @@
+# README
```

Comment:
might be worth noting setup steps like rails db:setup, I missed that one (and I should know better) :) It also wasn't immediately clear to me that I needed to create application.yml since there was an application.rb I thought maybe it was a typo. I'm assuming you are leaving it out of the repo on purpose since it has the client secret but you could potentially have the file and it assumes ENV variables or something like that.

Reply:
I updated the readme to be a little more explicit about the database setup.
The figaro gem I used will throw an error on startup if the keys don't have any values, so I figured that was just as clear as using environment variables.  I did take your suggestion and put an empty file in there.

Reply:
I didn't realize this, but git cannot ignore and track a file at the same time.  So instead I added an example config so nobody would be like me and accidentally check in their credentials (doh!).